### ISSUE TITLE
aot-eager accuracy error tracker

### ISSUE THREAD

**Issue by desertfire** [pytorch/torchdynamo#1272]
**HF training:**
- [x] #1273

**TIMM inference:**
- [x] #1274
- [x] pit_b_224

**TIMM training:**
- [x] coat_lite_mini
- [x] #1275
- [x] gmixer_24_224
- [x] mixnet_l
- [x] #1295
- [x] nfnet_l0
- [x] pit_b_224
- [x] tinynet_a
- [x] #1296

**Comment by eellison** [comment]
Hmm, I think these are all fixed on pytorch master. When you get a chance do you mind running just to make sure my setup up isn't wrong ?

**Comment by anijain2305** [comment]
I am closing this because many of these don't exist anymore. Let me track the remaining ones in https://github.com/pytorch/torchdynamo/issues/1091

**Comment by desertfire** [comment]
There are still a few remaining,  #1273, #1295, #1296.

**Comment by anijain2305** [comment]
Ah sorry, I did not try `amp` setting and closed prematurely. Thanks for tracking the `amp` failures as well.
